tests/legacy-cli/e2e/tests/misc/module-resolution.ts

```diff
@@ -12,10 +12,19 @@ export default async function () {
   });
   await ng('build');
 
+  await updateJsonFile('tsconfig.json', tsconfig => {
+    tsconfig.compilerOptions.paths = {
+      '@app/*': ['*'],
```

Comment:
Are these actually being create and consumed? I cannot find any reference to them anywhere.

I copied the paths from a test within the same file, but I have doubts they are actually correct.